### DISCUSSION TITLE
Eto Buttons

### DISCUSSION THREAD

**Discussion by shubhamjian** [picoe/Eto#1920] [Q&A]
Platform : Using Python on McNeel's Rhinoceros 3D Modelling software. 
Issue : I done want the border to show on my button, and if it is there I want to change its color, (but not able to find any documentation in support of background colour)
also **corners round radius is not dynamic,** like **i cannot change it, it is fixed value, i guess. Any procedure if i want to change these attributes how can i**

**Comment by cwensley** [comment]
If you want a control with custom appearance, you need to use the `Drawable` control and implement the button functionality that way, such as drawing via `Paint` event, mouse interaction via `MouseDown`, Keyboard interaction via `CanFocus`, `KeyDown`, etc.

Another option would be to create a [native style](https://github.com/picoe/Eto/wiki/Styling) to modify the WPF or AppKit Button's properties, depending on which platform it is running on.

Hope this helps!
